README.md

```diff
@@ -82,9 +60,7 @@ For testing purposes, we are using an internal web server by default at `http://
 The test server simulates some of the W3C services, such as the CSS and HTML validators, or the token authorization checker.
```

Comment:
Is this answered by what's proposed above (and potential modifications) or do you believe we need to change that as well?